src/blockstore/mod.rs

```diff
@@ -0,0 +1,38 @@
+mod api;
+#[cfg(not(target_arch = "wasm32"))]
+mod carv2_disk;
+mod carv2_memory;
+mod memory;
+#[cfg(not(target_arch = "wasm32"))]
+mod multi_carv2_disk;
+mod split;
+#[cfg(not(target_arch = "wasm32"))]
+#[cfg(test)]
+pub mod test;
+
+/// Api BlockStore
+pub use api::BanyanApiBlockStore;
+#[cfg(not(target_arch = "wasm32"))]
+pub use carv2_disk::CarV2DiskBlockStore;
+/// Memory based CarV2 formatted BlockStore implementation
+pub use carv2_memory::CarV2MemoryBlockStore;
+/// Disk based BlockStore implementation
```

Comment:
Remove redundant Comments + the Disk blockstore is no longer here, so remove that comment as well.